nemo/collections/nlp/nm/trainables/common/common_utils.py

```diff
@@ -70,6 +70,9 @@ def get_pretrained_lm_model(pretrained_model_name, config=None, vocab=None, chec
         raise ValueError(f'{pretrained_model_name} is not supported')
 
     if checkpoint:
-        model.restore_from(checkpoint)
+        if pretrained_model_name == 'megatron-bert-cased' or pretrained_model_name == 'megatron-bert-uncased':
+            model.megatron_restore_from(checkpoint, local_rank)
```

Comment:
this should be `if 'megatron' in pretrained_model_name:`

Reply:
Done.